### PR TITLE
1.0.4 patch train updates

### PR DESCRIPTION
**Pull request by JunTaoLuo**

### CODE REVIEW

**Comment by bricelam** [comment]
Yikes, we don't intend to ship this, do we?

File: src/ef/project.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.0.1",
```
Comment (JunTaoLuo) [inline src/ef/project.json:2]: Hmm, this might not be what is needed here.

**Comment by JunTaoLuo** [comment]
@bricelam The Microsoft.EntityFrameworkCore.Design package was updated in https://github.com/aspnet/EntityFramework/pull/7594 and that dependency made the updates here necessary. Though we are moving to MSBuild, we decided to still publish the project.json tools to be safe. cc @Eilon

**Comment by bricelam** [comment]
I hate that we've shipped the same buggy code over and over again and the newer one that people should actually be using is ordinally less than all of them.

**Comment by JunTaoLuo** [comment]
Closing now since it was decided to not build project.json tools.

**Pull request closed without merging**